docs/start/routing.md

```diff
@@ -256,28 +256,6 @@ You can destructure the `*`, you just have to assign it a new name. A common nam
 const { "*": splat } = params;
 ```
 
-## Case Sensitive Routes
-
-You can make your routes case sensitive by exporting a `config` object from `app/routes.ts`:
-
-```ts filename=app/routes.ts
-import {
-  defineRoutes,
-  route,
-} from "@react-router/dev/routes";
-
-export const config = {
-  caseSensitive: true,
-};
-
-export default defineRoutes([
```

Comment:
This use case doesn't currently exist, so I'm removing these docs rather than migrating them.